rebasehelper/tests/test_specfile.py

```diff
@@ -115,11 +140,6 @@ def test_get_patches(self, workdir, spec_object):
             patches[index] = [p.path, p.index]
         assert patches == expected_patches
 
-    def test_get_requires(self, spec_object):
-        expected = {'openssl-devel', 'pkgconfig', 'texinfo', 'gettext', 'autoconf'}
-        req = spec_object.header.requires
-        assert len(expected.intersection(req)) == len(expected)
-
```

Comment:
I agree that this test makes no sense, however, there is nothing about removal of meaningless/redundant tests in the commit message ;-)